File: app/src/main/java/com/stevenschoen/putionew/model/responses/FilesSearchResponse.java
package com.stevenschoen.putionew.model.responses;

import com.stevenschoen.putionew.model.ResponseOrError;
import com.stevenschoen.putionew.model.files.PutioFile;

import java.util.List;

public class FilesSearchResponse extends ResponseOrError.BasePutioResponse {
  public List<PutioFile> files;
}

File: app/src/main/java/com/stevenschoen/putionew/model/responses/FilesSearchResponse.kt
package com.stevenschoen.putionew.model.responses

import com.stevenschoen.putionew.model.ResponseOrError
import com.stevenschoen.putionew.model.files.PutioFile

class FilesSearchResponse(
    val files: List<PutioFile>
) : ResponseOrError.BasePutioResponse()
